orbs/helm/orb.yml

```diff
@@ -94,12 +98,12 @@ commands:
           command: |
             mkdir -p /tmp/helm-templates/<<parameters.environment>>
             helm template \
-              -n <<parameters.helm_release_name>> \
+              <<parameters.helm_release_name>> <<parameters.helm_chart_path>>
```

Comment:
**remark**
I think you might need a `\` at the end of this line no ? (and probably to remove the one at the end of line 106)

Reply:
Yep totally thx @happypoulp